Misc/NEWS.d/next/Library/2024-05-07-17-38-53.gh-issue-118714.XXKpVZ.rst

```diff
@@ -0,0 +1 @@
+Make the :mod:`pdb` post-mortem restart/quit message match the behavior
```

Comment:
Both behavior and message were changed. There were two cases involved, both under post-mortem debugging:

1. If the user does `restart`, it actually restarts the program, instead of raising an error and exit. (This is the behavior change).
2. If the user does `quit`, it quits the program *without* printing the line saying "we'll restart the program".

I think I should make the behavior change clearer (it's considered a bug fix because quitting the debugger when doing `restart` feels wrong).

Reply:
I forgot to change this after the review. Just changed it. Let me know if you think this could be better.